static/skin/index.js

```diff
@@ -14,18 +22,18 @@
     let isFetching = false;
     let noResultInjected = false;
     let filters = getCookie(filterCookieName);
-    let params = new URLSearchParams(window.location.search || filters || '');
+    let params = new FragmentParams(window.location.hash || filters || '');
     params.delete('userlang');
     let timer;
     let languages = {};
     let previousScrollTop = Infinity;
 
     function updateFeedLink() {
-        const inputParams = new URLSearchParams(window.location.search);
-        const filteredParams = new URLSearchParams();
+        const inputParams = new FragmentParams(window.location.hash);
+        const filteredParams = new FragmentParams();
         for (const [key, value] of inputParams) {
             if ( value != '' ) {
-                filteredParams.append(key, value);
+                filteredParams.set(key, value);
```

Comment:
I'm not sure about this change.
Why using set instead of append ?
I've the feeling this is a reminder a the previous version where you implement `FragmentParams`  using a map.

Reply:
Frankly, during my last review I also wanted to comment on this change, even wrote a comment but then deleted it, since this change has no impact on the meaningful URLs of the welcome page (there are no query params that can be repeated).

Reply:
Should I change this then? :>

Reply:
Let's move on and merge this PR !